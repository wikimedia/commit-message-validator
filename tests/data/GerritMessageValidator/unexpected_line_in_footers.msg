This is a commit subject

Commit body

Bug: T1234
Err: Unexpected line
Change-Id: If89d24838e326fe25fe867d02181eebcfbb0e196
