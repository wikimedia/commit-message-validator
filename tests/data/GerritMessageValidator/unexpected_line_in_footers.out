The following errors were found:
- Line 6: Unexpected footer 'Err'. Supported footers: %known_gerrit_footers%
